### PR TITLE
cfdyndns: fix broken build

### PR DESCRIPTION
**Pull request by voanhduy1512**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/networking/dyndns/cfdyndns/default.nix [hunk 1]
```diff
@@ -8,13 +8,13 @@ buildRustPackage rec {
...
-    sha256 = "1mcdjykrgh0jq6k6y664lai8sbgzk6j7k0r944f43vg63d1jql5b";
+    rev = "93221d18c42b6cdc766fa43c88bf11dbfc585d9d";
+    sha256 = "1q6mm586vn2928fb567yi09d0iclw2b51xh2kdzawzi2qpgc2vhs";
   };
 
-  cargoSha256 = "1d7jpffkw2m2v37bfdqsl9sqwsl19cgglpa00lwy4ih09kzbc2n9";
+  cargoSha256 = "0xa8w792rsyj1pb6i1rxsn2b1wk4bdm0kmbfhq7yzndfngbrapss";
 
-  buildInputs = [ makeWrapper openssl ];
+  buildInputs = [ makeWrapper openssl pkg-config];
```
Comment (worldofpeace) [inline pkgs/applications/networking/dyndns/cfdyndns/default.nix]: I'm finding it unlikely `makeWrapper` and `pkg-config` are runtime dependencies
Can you put these in `nativeBuildInputs`.

**Comment by worldofpeace** [comment]
The change in `nixos/modules/services/misc/cfdyndns.nix` is unrelated. Can you make this a different commit?

It also seems you've made this an update, but the commit doesn't reflect this being done.
Can you change `version` and have the commit message as standard in nixpkgs
```
cfdyndns: $old_version -> $new_version, fix build
```

File: pkgs/applications/networking/dyndns/cfdyndns/default.nix (hunk 1 above)
Comment (voanhduy1512) [reply pkgs/applications/networking/dyndns/cfdyndns/default.nix]: Sure can

**Comment by voanhduy1512** [comment]
The change in `nixos/modules/services/misc/cfdyndns.nix` is not related but the old 5 minutes syntax is not validate and systemd reject it. But sure i can move it to another commit.
Also cfdyndns haven't release a new version. I just update it to latest git commit to make it be able to build successfully, so not sure if i should ping cfdyndns side to ask them release new version and update this PR or put commit hash in the commit message?

**Comment by colemickens** [comment]
I cut a new tagged release: `v0.0.2` on cfdyndns in case it helps. I didn't push a release tarball or anything, but the tag is there so you can use it in this PR.

**Comment by worldofpeace** [comment]
This is stale

**Pull request closed without merging**